addons/l10n_ec/data/account_group_template_data.xml

```diff
@@ -0,0 +1,792 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
why noupdate = "1" and why not in csv (seems easier to read)

Reply:
easy to maintain for OPA > remove the noupdate

Reply:
Resolved